ca_on_toronto/people.py

```diff
@@ -27,15 +27,21 @@ def scrape(self):
     def scrape_councilor(self, page, h1, url):
         name = h1.split('Councillor')[1].strip()
         ward_full = page.xpath('//p/descendant-or-self::*[contains(text(), "Profile:")]/text()')[0].replace('\xa0', ' ')
-        ward_num, ward_name = re.search(r'(Ward \d+) (.+)', ward_full).groups()
+        ward_num, ward_name = re.search(r'Ward (\d+) (.+)', ward_full).groups()
         if ward_name == 'Etobicoke Lakeshore':
-            ward_name = 'Etobicoke\u2014Lakeshore'
+            ward_name = 'Etobicoke-Lakeshore'
 
-        district = '{0} ({1})'.format(ward_name.replace('-', '\u2014'), ward_num.split()[1])
+        ward_name = ward_name.replace('-', '\u2014')
```

Comment:
I know this is the legit way, but thoughts on using a hyphen to avoid unexpected user confusion down the road? I can just imagine being surprised by a lack of search results if search by ward names (in some interfaces). We can avoid that here, and just make sure we always normalize properly when scraping.